workflow for developing zkSNARK dApp by zokrates on bitcoin/README.md

```diff
@@ -0,0 +1,71 @@
+# 比特币上基于 zokrates 的 zkSNARK 应用开发流程简介
+
+
+我们展示了[如何使用 zk-SNARKs 来开发比特币的信息不完整的游戏](https://xiaohuiliu.medium.com/incomplete-information-games-on-bitcoin-d79408050882)。
+
+
+众所周知，[ZoKrates](https://github.com/Zokrates/ZoKrates) 是 zkSNARKs 的工具箱。它可以帮助您在 DApp 中使用可验证的计算，从规范的基于高级语言的程序电路到生成计算证明，再到验证这些证明。
+
+
+在 [ZoKrates Fork 库](https://github.com/sCrypt-Inc/zokrates) 中，我们扩展了 ZoKrates 的功能。使其支持自动生成验证 zk-SNARKs 证明的sCrypt智能合约。
```

Comment:
This ZoKrates fork is a toolbox for zkSNARKs on Bitcoin. It helps you use verifiable computation in your app, from the specification of your program in a high level language to generating proofs of computation to verifying those proofs in sCrypt.